src/main/api/2/studio-api-2.yaml

```diff
@@ -5339,6 +5339,28 @@ paths:
         '500':
           $ref: '#/components/responses/InternalServerError'
 
+  /security/preview/switch:
```

Comment:
I added it to Security controller because it already had the `AccessTokenService` dependency. It is also a security concern.

If this API is called with a `crafterSite` the user has no access to (or empty/absent), it will remove the `crafterPreview` cookie.
Response code would be 200.

Reply:
Should the response code 404 for a site that doesn't exist indicating that the site is not found (not found for this user)?

Reply:
Updated.
404 when user does not have access or site does not exist